File: 정수론/소수판별.py
def is_prime(num):
    if num == 1:
        return False
    if num % 2 == 0:
        return num == 2
    
    for i in range(3, int(num**0.5)+1, 2):
        if num % i == 0:
            return False
        
    return True

print(is_prime(17))  # True

File: 정수론/소수판별.js
const isPrime = (num) => {
  if (num <= 1) return false;
  if (num % 2 === 0) return num === 2;

  for (let i = 3; i <= Math.sqrt(num); i++) {
    if (num % i === 0) return false;
  }

  return true;
};

console.log(isPrime(17)); // true
